lib/helpers.js

```diff
@@ -329,6 +329,8 @@ module.exports = {
     }
 
     item.request = request;
+    // add name property, which postman will use to show in UI
+    item.name = request && request.url && request.url.path && request.url.path.join('/');
```

Comment:
Thanks @valentinkononov 💯 
Using _.get will be appropriate here
Also, can you add a regression test to the suite?

Reply:
ok, will do later this week.